### PR TITLE
add search bars

### PR DESCRIPTION
**Pull request by manch0ffline**
- [DEMO LINK](https://manch0ffline.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://manch0ffline.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,24 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <div class="search search--big" data-qa="big">
+      <div class="search--bigimage"></div>
```
Comment (VitaliyBodnar) [inline src/index.html:16]: Syntax mistake
```suggestion
      <div class="search--big-image"></div>
```

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,24 @@
...
+      <input type="text"
+        class="key key--big"
+        placeholder="Try “Los Angeles”"
+        data-qa="keypress"
+      >
+    </div>
+
+    <div class="search search--small" data-qa="small">
+      <div class="search--smallimage"></div>
```
Comment (VitaliyBodnar) [inline src/index.html:25]:
```suggestion
      <div class="search--small-image"></div>
```

File: src/style.css [hunk 3]
```diff
@@ -1 +1,103 @@
...
+  box-sizing: border-box;
+  align-items: center;
+  transition: box-shadow 0.3s ease;
+}
+
+.key {
+  outline: none;
+  border: none;
+  padding: 0;
+  font-family: Avenir-Book, sans-serif;
```
Comment (VitaliyBodnar) [inline src/style.css:45]: font-family is duplicated, you need to set it only once

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 4]
```diff
@@ -1 +1,102 @@
...
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf)
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
```
Comment (SanyaBratashchuk) [inline src/style.css:15]:
```suggestion
@font-face {
  font-family: Avenir;
  src: url(fonts/Avenir.ttc);
}

@font-face {
  font-family: Avenir;
  src: url(fonts/Avenir-Book.ttf)
}

@font-face {
  font-family: Avenir;
  src: url(fonts/Avenir-Heavy.ttf);
}
```
And just specify font-weight

File: src/style.css [hunk 5]
```diff
@@ -1 +1,102 @@
...
+
+.key {
+  outline: none;
+  border: none;
+  padding: 0;
+}
+
+.key::placeholder {
+  color: var(--text-color);
+  font-family: Avenir-Book, sans-serif;
```
Comment (SanyaBratashchuk) [inline src/style.css:49]: apply font-family for `html` tag selector and just change font-weight

File: src/style.css [hunk 6]
```diff
@@ -1 +1,102 @@
...
+  border-radius: 4px;
+  border: 1px solid var(--border-bars);
+  background: var(--bg-page);
+  box-shadow: var(--shadow-bars);
+  box-sizing: border-box;
+  align-items: center;
+  transition: box-shadow 0.3s ease;
+}
+
+.key {
```
Comment (SanyaBratashchuk) [inline src/style.css:41]:
```suggestion
.key {
font-family: inherit;
```

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 7]
```diff
@@ -11,6 +11,24 @@
...
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <div class="search search--big" data-qa="big">
+      <div class="search--big-image"></div>
+      <input type="text"
+        class="key key--big"
+        placeholder="Try “Los Angeles”"
+        data-qa="keypress"
+      >
```
Comment (anastasiiavorobiova) [inline src/index.html:21]:
```suggestion
      <input 
        type="text"
        class="key key--big"
        placeholder="Try “Los Angeles”"
        data-qa="keypress"
      >
```

File: src/index.html [hunk 8]
```diff
@@ -11,6 +11,24 @@
...
+    </div>
+
+    <div class="search search--small" data-qa="small">
+      <div class="search--small-image"></div>
+      <input type="text"
+        class="key key--small"
+        placeholder="Try “Los Angeles”"
+        data-qa="keypress"
+      >
```
Comment (anastasiiavorobiova) [inline src/index.html:30]:
```suggestion
      <input 
        type="text"
        class="key key--small"
        placeholder="Try “Los Angeles”"
        data-qa="keypress"
      >
```

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Consider making the whole area of the input clickable 
Screenshot 2023-11-26 at 19 31 22

**Pull request closed without merging**